medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/DocumentController.cs

```diff
@@ -175,5 +175,63 @@ public async Task<IActionResult> UploadDriverDocument([FromForm] IFormFile file,
         }
 
 
+        [HttpPost("claimDmer")]
+        [ProducesResponseType(typeof(CaseDocument), 200)]
+        [ProducesResponseType(401)]
+        [ProducesResponseType(500)]
+        public async Task<IActionResult> UpdateClaimDmerOnDocument([FromRoute] string documentId)
+        {
+            var profile = await _userService.GetCurrentUserContext();
+            var loginId = profile.LoginId;
+
+            var request = new UpdateClaimRequest { 
+                LoginId = loginId,
+                DocumentId = documentId
+            };
+            
+            var reply = _documentManagerClient.UpdateClaimDmer(request);
+            if (reply.ResultStatus == Rsbc.Dmf.CaseManagement.Service.ResultStatus.Success)
+            {
+                var caseDocuments = _mapper.Map<CaseDocument>(reply.Item);
+
+                return Ok(caseDocuments);
+            }
+            else
+            {
+                _logger.LogError($"{nameof(UpdateClaimDmerOnDocument)} error: unable to Claim DMER document - {reply.ErrorDetail}");
+                return StatusCode(500, reply.ErrorDetail);
+            }
+        }
+
+
+        [HttpPost("unclaimDmer")]
+        [ProducesResponseType(typeof(CaseDocument), 200)]
+        [ProducesResponseType(401)]
+        [ProducesResponseType(500)]
+        public async Task<IActionResult> UpdateUnclaimDmerOnDocument([FromRoute] string documentId)
+        {
+            var profile = await _userService.GetCurrentUserContext();
+            var loginId = profile.LoginId;
+
+            var request = new UpdateClaimRequest
+            {
+                LoginId = loginId,
+                DocumentId = documentId
+            };
+
+            var reply = _documentManagerClient.UpdateClaimDmer(request);
```

Comment:
is this right? The service is Unclaim and this is Claim

Reply:
Fixed the method to use Unclaim DMER